main/guides/zoe/contracts/constantProductAMM.md

```diff
@@ -49,7 +52,7 @@ any trading has intervened, the trade is unlikely to be accepted. You can either
 specify limits on how far the price may have moved, or specify limits of zero and
 trust the contract to trade fairly.
 
-Transactions that don't require an invitation include `addPool()` and the queries
+Transactions that don't require an invitation include `addPoolInvitation()` and the queries
```

Comment:
I don't see `addPoolInvitation` in the codebase https://github.com/search?q=repo%3AAgoric%2Fagoric-sdk%20addPoolInvation&type=code